lib/comgr/include/amd_comgr.h

```diff
@@ -75,8 +75,15 @@
 #endif
 #endif
 
+#ifndef AMD_COMGR_INTERFACE_VERSION_MAJOR
 #define AMD_COMGR_INTERFACE_VERSION_MAJOR 1
+#endif
+#ifndef AMD_COMGR_INTERFACE_VERSION_MINOR
 #define AMD_COMGR_INTERFACE_VERSION_MINOR 6
+#endif
+#ifndef AMD_COMGR_INTERFACE_VERSION_PATCH
+#define AMD_COMGR_INTERFACE_VERSION_PATCH 0
+#endif
```

Comment:
As it stands defining the `_PATCH` here seems like dead code. The API for querying the major+minor version numbers may even be gratuitous now that we are starting to version the .so

Reply:
Yes, for non cmake builds error will be shown. I will remove this code part and update the commit.